<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="com.org.iot.test.smoke.Smoke" tests="10" time="29.239" ignored="0" failures="1" timestamp="08 May 2017 13:28:42 GMT" skipped="0" hostname="RAJUK4" errors="0">
  <testcase name="updateAE" time="2.328" classname="com.org.iot.test.smoke.Smoke"/>
  <testcase name="getContainerData" time="0.125" classname="com.org.iot.test.smoke.Smoke"/>
  <testcase name="createSubscriptionMetadata" time="0.432" classname="com.org.iot.test.smoke.Smoke">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.fail(Assert.java:101)
	at com.org.iot.test.smoke.Smoke.createSubscriptionMetadata(Smoke.java:807)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </failure>
  </testcase> <!-- createSubscriptionMetadata -->
  <testcase name="updateCNT" time="0.876" classname="com.org.iot.test.smoke.Smoke"/>
  <testcase name="createGroup" time="10.606" classname="com.org.iot.test.smoke.Smoke"/>
  <testcase name="createSubscriptionCNT" time="2.977" classname="com.org.iot.test.smoke.Smoke"/>
  <testcase name="createContainer" time="8.622" classname="com.org.iot.test.smoke.Smoke"/>
  <testcase name="createSubscriptionAE" time="0.474" classname="com.org.iot.test.smoke.Smoke"/>
  <testcase name="getAE" time="2.134" classname="com.org.iot.test.smoke.Smoke"/>
  <testcase name="createSubscriptionGroup" time="0.665" classname="com.org.iot.test.smoke.Smoke"/>
</testsuite> <!-- com.org.iot.test.smoke.Smoke -->
